pyTigerGraph/pytgasync/pyTigerGraphAuth.py

```diff
@@ -0,0 +1,201 @@
+"""Authentication Functions
+
+The functions on this page authenticate connections and manage TigerGraph credentials.
+All functions in this module are called as methods on a link:https://docs.tigergraph.com/pytigergraph/current/core-functions/base[`TigerGraphConnection` object].
+"""
+
+import logging
+from typing import Union
+import httpx
+
+from pyTigerGraph.pyTigerGraphException import TigerGraphException
+from pyTigerGraph.pyTigerGraphAuth import pyTigerGraphAuth
+from pyTigerGraph.pytgasync.pyTigerGraphGSQL import AsyncPyTigerGraphGSQL
+
+logger = logging.getLogger(__name__)
+
+# TODO FINISH MAKING THE ASYNCS AND ADD COMMENTS
```

Comment:
Is this todo handled?

Reply:
This looks like old TODO's. Let me remove it.